Rakefile

```diff
@@ -139,6 +147,11 @@ begin
       title 'Running examples'
       Rake::Task['examples:build'].invoke
 
+      if true || ENV['TRAVIS_PULL_REQUEST'] == 'false'
```

Comment:
yeah, we're trying to make sure this PR actually passes with the regression tests running before merging